mohawk/bewit.py

```diff
@@ -48,7 +48,7 @@ def get_bewit(resource):
     # Oct 28, 2015, so this could break compat.
     # We can leave \ in ext since validators can limit how many \ they split
     # on (although again, the canonical implementation does not do this)
-    client_id = resource.credentials['id']
+    client_id = six.text_type(resource.credentials['id'])
```

Comment:
Did you try creating a failing test for this? I'm surprised it wasn't failing in 3.5 already.